src/main/java/de/retest/recheck/auth/AuthenticationHandler.java

```diff
@@ -6,8 +6,10 @@
 
 	void showWebLoginUri( URI loginUri );
 
-	void authenticated();
+	void loginPerformed( String token );
```

Comment:
Do you have a branch in place for that in downstream projects (read review)?

Reply:
Yes, I do.